[
    {
        "repo_name": "upx/upx",
        "CVE_ID": "CVE-2021-30501",
        "Problem_Type": "CWE-617",
        "Description": "An assertion abort was found in upx MemBuffer::alloc() in mem.cpp, in version UPX 4.0.0. The flow allows attackers to cause a denial of service (abort) via a crafted file.",
        "URL": "https://github.com/upx/upx/pull/487",
        "Tag": "['Exploit', 'Patch', 'Third Party Advisory']",
        "fix_commit_hash": null
    }
]